avm/res/service-bus/namespace/main.bicep

```
@@ -379,6 +379,7 @@ resource serviceBusNamespace_diagnosticSettings 'Microsoft.Insights/diagnosticSe
 
 module serviceBusNamespace_privateEndpoints 'br/public:avm/res/network/private-endpoint:0.10.1' = [
   for (privateEndpoint, index) in (privateEndpoints ?? []): {
+    #disable-next-line BCP335
     name: '${uniqueString(deployment().name, location)}-serviceBusNamespace-PrivateEndpoint-${index}'
```

Alternatively you could shorten the deployment name, e.g. like this

```suggestion
    name: '${uniqueString(deployment().name, location)}-serviceBusNamespace-PE-${index}'
```

---

Good point , thanks! Decided to abbreviate to 'PEP' instead of 'PE' in accordance with MS abbreviation recommendations :)